pkgs/applications/version-management/smartgithg/default.nix

```diff
@@ -54,6 +54,7 @@ stdenv.mkDerivation rec {
       --prefix SMARTGITHG_JAVA_HOME : ${jre}
     sed -i '/ --login/d' ${pkg_path}/bin/smartgit.sh
     patchShebangs $out
+    wrapGAppsHook
```

Comment:
This is ran automatically on the binaries. However in this case we will have two wrappers created by `makeWrapper`

We can work around this by setting `dontWrapGApps = true;` and using the array of arguments
it would pass to `wrapProgram` manually in the `makeWrapper` call above.

See
* https://github.com/NixOS/nixpkgs/blob/10c103614f3074877337d4627ff1b0a1f4984864/pkgs/applications/editors/typora/default.nix#L44

for an example.

Reply:
I've changed this to use wrapGappsHook and append to gappsWrapperArgs instead of using `makeWrapper`.